doc/source/report/dynamics.rst

```diff
@@ -6,114 +6,53 @@ Dynamics of AS Patterns
 Statement of Problem
 --------------------
 
-The objective of Dynamics Analysis is to classify the active states and
-inactive states, as well as understand the dynamics/transitions between
-AS and IS..
+The objective of Dynamics Analysis is to analyze and characterize the behaviors of different strains of mice using Markov Chain model. We could use the MC models to further explore the discrepancy between different strains and even simulate a particular mouse day for a particular strain.
```

Comment:
line 9 can be paraphrased as below because the meaning is a little unclear:
The objective of Dynamics Analysis is to analyze the behaviors of different strains of mice by using Markov Chain model. The MC model can be used further to explore the discrepancy between different strains, and it even can be used to simulate a particular mouse day for a particular strain.